tests/test__mongomock.py

```diff
@@ -131,6 +131,18 @@ def test__drop_database_indexes(self):
         result = collection.find({})
         self.assertEqual(result.count(), 3)
 
+    def test__sparse_unique_index(self):
+        db = self.client.somedb
+        collection = db.a
+        collection.create_index('simple')
```

Comment:
Please drop that line, the test works without it, and it could lead to confusion.

Reply:
ok